scripts/pr-audit.sh

```
@@ -0,0 +1,972 @@
+import json, sys
+d = json.load(sys.stdin)
+runs = d.get('check_runs', [])
+total = d.get('total_count', 0)
+in_progress = sum(1 for r in runs if r.get('status') == 'in_progress')
+failed = sum(1 for r in runs if r.get('conclusion') in ('failure', 'cancelled', 'timed_out'))
+succeeded = sum(1 for r in runs if r.get('conclusion') == 'success')
+print(f'check_total={total} check_in_progress={in_progress} check_failed={failed} check_succeeded={succeeded}')
+" 2>/dev/null || echo "check_total=0 check_in_progress=0 check_failed=0 check_succeeded=0")
+
+eval "$check_run_analysis"
+check_run_failures=$check_failed
```

Shellcheck (0.11.0)

[warning] check_failed is referenced but not assigned.

(SC2154)